docs/using/serialization.md

```diff
@@ -0,0 +1,41 @@
+# Serialization
+
+`Interchange` objects can be written to disk and read back at a later date (or on another machine, or by another person). Under the hood, serialization and de-serialization are largely handled by Pydantic, whose building block `pydantic.BaseModel` is a base class of `Interchange`. The serialization API is therefore inherited from Pydantic. It uses a mix of Python and Rust, so it's relatively fast.
```

Comment:
```suggestion
`Interchange` objects can be written to disk in JSON format and read back into an Interchange at a later date (or on another machine, or by another person). Under the hood, serialization and de-serialization are largely handled by Pydantic, whose building block `pydantic.BaseModel` is a base class of `Interchange`. The serialization API is therefore inherited from Pydantic. It uses a mix of Python and Rust, so it's relatively fast.
```

I think it's useful to mention the JSON format early on, just to give some more context on what's going on and why exporting to GROMACS/Amber/LAMMPS is different. I also think the phrasing of "read back" could be improved to make it clearer that you're getting an unchanged Interchange back at the end?